index.html

```diff
@@ -0,0 +1,319 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Udemy</title>
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/4b9cd3dbd3.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <div class="main-content" id="">
+      <header>
+        <div class="top-header">
+          <div class="bar">
+            <div class="text">
+              <span class="presentation">
+                <span class="title">Courses on sale from E£169.99</span>
+                <span>
+                  | Thanks for enrolling in a free course. Now, upgrade your
+                  learning.
+                </span>
+              </span>
+              <span class="un-title"> Ends in 5h 52m 34s.</span>
+            </div>
+          </div>
+          <button class="remove-text">x</button>
```

Comment:
I think this should be `btn-remove` or `btn-close`, right?

Reply:
Done